website/docs/docs/build/exposures.md

```diff
@@ -24,6 +24,39 @@ Exposures make it possible to define and describe a downstream use of your dbt p
 
 Exposures are defined in `.yml` files nested under an `exposures:` key.
 
+<VersionBlock firstVersion="1.4">
+
+<File name='models/<filename>.yml'>
+
+```yaml
+version: 2
+
+exposures:
+
+  - name: weekly_jaffle_metrics
+    type: dashboard
+    maturity: high
+    url: https://bi.tool/dashboards/1
+    description: >
+      Did someone say "exponential growth"?
+
+    depends_on:
+      - ref('fct_orders')
+      - ref('dim_customers')
+      - source('gsheets', 'goals')
+      - metric('example_metric_name')
```

Comment:
Instead of `example_metric_name`, let's use [`count_orders`](https://github.com/dbt-labs/dbt-docs/blob/main/ci-project/models/metrics.yml#L5).

---

_big aside_

When writing pseudo code / example code, I try to coordinate all the code to use either:
1. "real" (imagined) names — most immersive, suggesting a potential use case
2. placeholder names (`example_metric_name`, `my_model`) — clearest & most flexible, though lacking the "why"
3. data structure (`<model_name: str>`) — most technical & informative, less accessible to less experienced users

This example uses "real" (imagined) names for models/sources/etc, loosely based on [`jaffle_shop`](https://github.com/dbt-labs/jaffle_shop) / the sample project [here](https://github.com/dbt-labs/dbt-docs/tree/main/ci-project).